extensions/azurecore/package.json

```diff
@@ -41,6 +41,39 @@
             "description": "%config.enablePublicCloudDescription%"
           }
         }
+      },
+      {
+        "type": "object",
```

Comment:
You can just combine these all into the existing Azure Account Configuration (just add new property objects). See https://github.com/microsoft/azuredatastudio/blob/master/extensions/machine-learning-services/package.json#L32 for an example